utils/envvars/parser.go

```diff
@@ -36,8 +36,10 @@ const (
 	envSaaSHost     = "PERCONA_TEST_SAAS_HOST"
 	envPublicKey    = "PERCONA_TEST_CHECKS_PUBLIC_KEY"
 	// TODO REMOVE PERCONA_TEST_DBAAS IN FUTURE RELEASES.
-	envTestDbaas   = "PERCONA_TEST_DBAAS"
-	envEnableDbaas = "ENABLE_DBAAS"
+	envTestDbaas              = "PERCONA_TEST_DBAAS"
+	envEnableDbaas            = "ENABLE_DBAAS"
+	envPlatfromAPITimeout     = "PERCONA_TEST_PLATFORM_API_TIMEOUT"
```

Comment:
```suggestion
	envPlatfromAPITimeout     = "PERCONA_PLATFORM_API_TIMEOUT"
```

this is not for testing but may be used in production as well by users

Reply:
Done